scipy/stats/_multivariate.py

```diff
@@ -786,6 +786,36 @@ def entropy(self, mean=None, cov=1):
         dim, mean, cov_object = self._process_parameters(mean, cov)
         return 0.5 * (cov_object.rank * (_LOG_2PI + 1) + cov_object.log_pdet)
 
+    def fit(self, x):
+        """Fit the multivariate normal to data `x`.
+
+        Parameters
+        ----------
+        x : data the distribution is fitted to. The last axis determines the
```

Comment:
[In our docstring format](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), the thing following the `:` is a type. The prose description of the parameter follows indented. [Same for the `Returns`](https://numpydoc.readthedocs.io/en/latest/format.html#returns).

```
x : array_like
    The data the distribution is fitted to...
```